composer/trainer/trainer.py

```diff
@@ -141,8 +141,12 @@ def _set_evaluator_interval_and_subset_num_batches(
             evaluator.eval_interval = eval_interval
 
 
-def _is_adaptive_grad_accum(grad_accum: Union[int, str], device: Device):
+def _is_adaptive_grad_accum(grad_accum: Union[int, str], device: Device, profiler: Optional[Profiler]):
     if grad_accum == 'auto':
```

Comment:
[Optional]: Though it is not necessary as part of this PR, but just wondering to add a method comments to understand the code better. Thanks!